Resources/Prototypes/Entities/Structures/Furniture/instruments.yml

```diff
@@ -31,14 +31,19 @@
       type: InstrumentBoundUserInterface
   - type: Fixtures
     fixtures:
-      - shape:
-          !type:PhysShapeCircle
-          radius: 0.45
-        mass: 50
-        layer:
-          - SmallImpassable
-        mask:
-          - VaultImpassable
+    - shape:
+        !type:PhysShapeCircle
+        radius: 0.45
+      mass: 50
+      mask:
+      - Impassable
+      - HighImpassable
+      - MidImpassable
+      layer:
+      - Opaque
+      - HighImpassable
+      - MidImpassable
+      - BulletImpassable
```

Comment:
if this is for placeable instruments (DAW?) should this just be `TabletopMachineMask/Layer`?

Reply:
I was going off of what would fit on Pianos, Xylophones, and Minimoogs which are full tile and can't really put stuff on top of them.